www/%team/index.html.spt

```diff
@@ -18,7 +18,9 @@ is_team_owner = not user.ANON and team.owner == user.participant.username
 {% block head %}
     <link rel="payment" type="text/html" title="Give weekly gift on Gratipay"
           href="https://gratipay.com/{{ team.slug }}/" />
+    <meta name="ROBOTS" content="NOINDEX, NOFOLLOW" />
```

Comment:
Under the [right to be forgotten](https://en.wikipedia.org/wiki/Right_to_be_forgotten), ~~teams~~ projects should have control over their own information. We leave it up as an example, which I assume falls under the TOS. An option might be to (somehow) anonymize rejected ~~teams~~ projects...but this seems like an easier solution unless someone yells. It's also possible for the owner to edit most of the information, although the `slug` remains.

On this note, are we going to re-review ~~teams~~ projects that were previously rejected once [Decouple](https://github.com/gratipay/gratipay.com/milestone/22) is complete?